src/style.css

```diff
@@ -1,3 +1,72 @@
 body {
   margin: 0;
+  font-family: Roboto, Arial, sans-serif, Helvetica;
```

Comment:
You should add a fallback font in case the main one doesn't work. This is a good practice to ensure that your website has a similar look on different devices and browsers. For example, you can use a generic font family like 'sans-serif' as a fallback.